src/test_l1_tvSettings.c

```diff
@@ -12580,7 +12580,7 @@ void test_l1_tvSettings_negative_GetDefaultGammaTable (void)
         platformFlag = false;
         for (unsigned int j = 0; j < colorTempCount ; j++ )
         {
-            snprintf(keyValue, UT_KVP_MAX_ELEMENT_SIZE, "tvSettings/ColorTemperature/index/%d", i);
+            snprintf(keyValue, UT_KVP_MAX_ELEMENT_SIZE, "tvSettings/ColorTemperature/index/%d", j);
             colorTempValue = (tvColorTemp_t) UT_KVP_PROFILE_GET_UINT32(keyValue);
             if(colorTempValue == (tvColorTemp_t)i)
             {
```

Comment:
In line number 12593, are we supposed to pass the index i instead of colorTempValue? This is because we intend to pass the unsupported color temp enum value.

Reply:
updated